### PR TITLE
Fix images validation bundle issue

### PR DESCRIPTION
**Pull request by cooler333**
fix issue #422

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ project 'ResourceApp/ResourceApp'
 abstract_target 'Shared' do
   inhibit_all_warnings!
 
-  pod 'R.swift.Library', :git => 'git@github.com:mac-cain13/R.swift.Library.git' # for CI builds
```
Comment (cooler333) [inline Podfile]: ssh -> https

File: Podfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ project 'ResourceApp/ResourceApp'
...
+  pod 'R.swift.Library', :git => 'https://github.com/mac-cain13/R.swift.Library.git' # for CI builds
```
Comment (cooler333) [inline Podfile]: ssh -> https

**Comment by Yuop** [comment]
nice one!

**Comment by YuryBogdanov** [comment]
@mac-cain13 can we somehow merge this?

**Comment by tomlokhorst** [comment]
Thank you for this PR!

This fix will be available in the next release.

**Pull request merged**